specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/CommonDefinitions.json

```diff
@@ -952,6 +952,93 @@
         }
       }
     },
+    "SwiftVirtualNetwork": {
+      "description": "Swift Virtual Network Contract",
+      "type": "object",
+      "properties": {
+        "subnetResourceId": {
+          "description": "The Virtual Network subnet's ARM ID",
+          "type": "string"
+        },
+        "swiftSupported": {
+          "description": "Boolean flag indicating whether Swift integration is supported",
+          "type": "boolean"
+        },
+        "subnetIpAllocation": {
+          "description": "IP allocation for a Swift Subnet",
+          "type": "object",
+          "properties": {
+            "subnetIpAddressesAvailable": {
+              "description": "The number of IP addresses available in the Swift subnet address space",
+              "type": "integer",
+              "format": "int32"
+            },
+            "subnetIpAddressesUsed": {
+              "description": "The number of IP addresses that have already been used in the Swift subnet address space",
+              "type": "integer",
+              "format": "int32"
+            }
+          }
+        },
+        "vnetConnectionAllocation": {
```

Comment:
>vnetConnectionAllocation [](http://example.com/codeflow?start=9&length=24)

Suggest using consistent naming for properties of subnetIpAllocation and vnetConnectionAllocation, e.g. use "available" or "max", not both.

Reply:
These properties are already in use and we can not rename without breaking customers. The naming difference is also deliberate and the reason for the difference is, that "max" refers to limits specified by the platform/product group, where "available" refers to limits based on the resources customers use - in this case the address range size of the subnet used and can therefore change.